plugins/janus_streaming.c

```diff
@@ -6519,6 +6521,23 @@ static int janus_streaming_rtsp_connect_to_server(janus_streaming_mountpoint *mp
 	}
 	long code = 0;
 	res = curl_easy_getinfo(curl, CURLINFO_RESPONSE_CODE, &code);
+#if CURL_AT_LEAST_VERSION(7, 62, 0)
+	if(code == 404) { //Possibly a quirk in the RTSP server, where the DESCRIBE request expects a path only.
```

Comment:
Just a nit, per code style we only use `/* */` for comments: I'd also put it as the first line inside the if, rather than on the same line as the if itself (just for the sake of readability).